java/test/jmri/jmrit/display/layoutEditor/BitsOfJavaTest.java

```diff
@@ -0,0 +1,26 @@
+package jmri.jmrit.display.layoutEditor;
```

Comment:
Is the intention to add more stuff to this class? It seems that it doesn't do anything useful now. I have no problem with it, just checking.

Reply:
Thanks, looks like this was committed in error. I've rm'd it.